v2_migrations/2024-08-28-081721_add_v2_columns/down.sql

```diff
@@ -43,3 +43,6 @@ ALTER TABLE payment_attempt DROP COLUMN payment_method_type_v2,
     DROP COLUMN payment_method_billing_address,
     DROP COLUMN redirection_data,
     DROP COLUMN connector_payment_data;
+
+ALTER TABLE merchant_connector_account
+ALTER COLUMN payment_methods_enabled TYPE JSON [ ];
```

Comment:
We should use JSONB Array instead of JSON Array

Reply:
This is in down.sql, this was the previous type, I have changed the type to JSONB [ ] in up.sql, so in down.sql had to change it back